src/pages/_app.tsx

```diff
@@ -9,25 +9,22 @@ import { Seo } from 'components/layout/Seo'
 
 export default function App({ Component, pageProps }: AppProps) {
   const isMounted = useIsMounted()
-  const desiredChainId = ChainId.Polygon
 
   // Create a client
   const queryClient = new QueryClient()
 
   return (
-    <ThirdwebProvider desiredChainId={desiredChainId}>
-      <ChakraProvider>
-        <Seo />
-        <Web3Provider>
-          {isMounted && (
-            <Layout>
-              <QueryClientProvider client={queryClient}>
-                <Component {...pageProps} />
-              </QueryClientProvider>
-            </Layout>
-          )}
-        </Web3Provider>
-      </ChakraProvider>
-    </ThirdwebProvider>
+    <ChakraProvider>
+      <Seo />
+      <Web3Provider>
+        {isMounted && (
+          <Layout>
+            <QueryClientProvider client={queryClient}>
+              <Component {...pageProps} />
+            </QueryClientProvider>
+          </Layout>
```

Comment:
`SignInButton` is in `Header` which is in `Layout`, but `Layout` is not a child of `QueryClientProvider`

```suggestion
          <QueryClientProvider client={queryClient}>
            <Layout>
              <Component {...pageProps} />
            </Layout>
          </QueryClientProvider>
```